tests/bin/bootstrap.test.ts

```diff
@@ -2,15 +2,17 @@ import os from 'os';
 import path from 'path';
 import fs from 'fs';
 
-import { checkKeynodeState } from '@/bootstrap';
+import { isDirectoryClear } from '@/bootstrap';
```

Comment:
It seems like this file should be a set of utility functions like `src/bootstrap/utils.ts`. They are just a bunch of different functions.